src/store/lokapi.ts

```diff
@@ -33,21 +33,31 @@ export var moduleLokAPI = {
         commit('auth_error')
         throw err
       }
-      await dispatch('setBackends')
-
-      commit("setThisWeekTransactions")
       commit('auth_success')
     },
     async resetTRS({commit} :any) {
-      await commit("setThisWeekTransactions")
+      commit("setTransactions", [])
     },
     async initAutoLogin({commit, dispatch}:any) {
-      commit("autoLogin")
-      await dispatch('setBackends')
+      commit('auth_success')
+      dispatch("fetchUserProfile")
+      dispatch("fetchBankAccounts")
+      dispatch("fetchBackends")
+      dispatch("fetchTransactions")
```

Comment:
I don't feel comfortable to make `autoLogin` search again everything from the app, and it make this very expensive. We might not have in mind the same purpose for this function. I'm not sure it is relevant here, but at some point, the auto-login could be about only fetching what is necessary to get either odoo/cyclos connection working again (it's about broken session tokens). I understand that it is not called for that currently, but shouldn't the components trigger themselves the differents data query when they need it ?  I guess that the previous code had this in mind, (and was asking for `backends` and only backends because this is about setting back the odoo/cyclos connection tokens and allowing any query to work.)

Reply:
Really ? I genuinely thought autologin was about re-fetching user profile and:
- bank accounts
- backends
- transactions
from the Odoo backend...
Because, from what I see of the app, when we are on the landing screen ,which is called profile in your taxonomy, we need all the bank accounts so that we display them, we need all the transactions so that we display them. Maybe the only thing we do not need is the backends, since it's needed to create the bank accounts. But even this I consider basic data that should be loaded from the start. If this is very expensive as you put it, then I think there is a problem elsewhere. Maybe these calls are fetching too much data at the same time ?

At this point I'm a bit lost:
> and was asking for backends and only backends because this is about setting back the odoo/cyclos connection tokens and allowing any query to work.
I don't even know how I was supposed to know that since it is written nowhere and it's the first time I encounter such a pattern. We should have a talk about this too. I need some info.

We should take some time so you can explain to me how all the monujo/lokapi is designed, because I really do not have a good grasp of it it seems.